data/table.txt

```diff
@@ -8096,7 +8146,8 @@
 勖 勖 1 1 0 0 1 0 0 0 0 auks auks NA 11602
 勗 勗 1 1 0 0 1 0 0 0 0 abms abms NA 19737
 勘 勘 1 1 0 0 1 0 0 0 0 tvks tvks NA 20021
-務 务 1 1 0 0 1 0 0 0 0 nhoks nhhes,nhoks NA 20022
+務 务 1 1 0 0 1 0 0 0 0 nhoks nhhes NA 20022
+務 务 1 1 0 0 1 0 0 0 0 NA nhoks NA 0
```

Comment:
@yookoala [said](https://github.com/Cangjians/libcangjie/commit/d9d2e686ce6776ec67e89b4256dce42359597d84#r29099647):

> "nhoks" is the standard mapping on MacOS. "nhhes" returns nothing.

---

Should we remove `nhhes` before running the script, then?

Reply:
So Wan Leung just said that Mac OS X uses the short-lived Cangjie version 4, not version 5. :open_mouth:

Discussing this with @yookoala, we decided to ignore all those comments about Mac.